.github/workflows/build-release.yml

```diff
@@ -0,0 +1,36 @@
+name: replication-backend-release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build_and_publish:
+    name: Build and publish replication-backend package
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Login to GitHub Container Registry
```

Comment:
If we switch to GitHub Container Registry from DockerHub, let's check all docs (espcially ndb-setup repo) to update links.

Reply:
Yes, i will update all the docs.